src/providers/live/live-provider.ts

```diff
@@ -32,7 +32,7 @@ export class LiveProvider extends Provider {
     this._currentTimeLivePromise = this._makeCurrentTimeLiveReadyPromise();
     this._pushNotification.init();
     this._constructPushNotificationListener();
-    this._pushNotification.registerToPushServer(this._player.getMediaInfo().entryId, types, this._handleConnection, this._handleConnectionError);
+    this._pushNotification.registerToPushServer(this._player.config.sources.id, types, this._handleConnection, this._handleConnectionError);
```

Comment:
we are actually planning to remove the sources form config one day so better not use sources too widely in our code.
You can use player.sources instead.

Reply:
done